### DISCUSSION TITLE
Dashboard Customization

### DISCUSSION THREAD

**Discussion by benrabicoff** [AdguardTeam/AdGuardHome#6381] [Ideas]
I think it would be a great feature to have more dashboard customization, particularly around hiding/showing/reordering of sections, and being able to have those sections expand larger vertically.

It would also be great to have the ability of assigning icons to clients for a better visual cue, making the client list far easier to grok. And something similar with domains/upstreams — perhaps a favicon could be pulled in.

I'm doing this all currently with a custom Safari style sheet that runs on startup, and icons base64 encoded based on targeting the client/domain/upstream name with css, ex `.dashboard .logs__row:has(.logs__text[title*="AppleTV"]):before {background-image: var(--icon-apple-tv) !important;}`

The lift may be heavier/more complex for the icons, but it would be great to have this all a part of platform itself! Cheers!
 
![dash](https://github.com/AdguardTeam/AdGuardHome/assets/447708/0cac40c5-6df4-4bad-87df-836e8d15994e)

**Comment by lutfor-diu** [comment]
wow. i love those icon and favicon. looks more modern and slicker! how to do it?
